wezterm-gui/src/customglyph.rs

```diff
@@ -220,6 +192,321 @@ pub enum BlockKey {
     },
 }
 
+// `pattern` is a byte whose bits corresponds to elements on a 2 by 3 grid.
+// The position of a sextant for a bit position (0-indexed) is as follows:
+// ╭───┬───╮
+// │ 0 │ 1 │
+// ├───┼───┤
+// │ 2 │ 3 │
+// ├───┼───┤
+// │ 4 │ 5 │
+// ╰───┴───╯
+const SEXTANT_PATTERNS: [u8; 60] = [
```

Comment:
Hello! Thanks for making the changes, this sextant sequences looks almost perfectly like a continuous sequence, only skipping `0b101010`.
In my opinion I'd just impl this as a perfect sequence, implementing even the one that is not defined in spec 🤔

I'm not the one who's going to review this in the end, that's @wez' thing, I'm just a 'pre-processor' here. 

nice job anyway 👍

Reply:
This array is the mapping from code points to sextant patterns. It's fixed by the standard – I couldn't just add the four missing ones.

The code that takes the bit pattern and draws the sextants is generic. It works for all 64 possible patterns although only 60 of them will ever be encountered.